### PR TITLE
Added Microsoft.ApiManagement Schemas

### PR DESCRIPTION
**Pull request by solankisamir**
- Added Version 2016-07-07
- Added tests for entities registered in ARM

### CODE REVIEW

File: tests/2016-07-07/Microsoft.ApiManagement.tests.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  "tests": [
+    {
+      "name": "ApiManagement service - Minimal",
+      "definition": "http://schema.management.azure.com/schemas/2016-07-07/Microsoft.ApiManagement.json#/resourceDefinitions/service",
+      "json": {
+        "type": "Microsoft.ApiManagement/service",
+        "apiVersion": "2016-07-07",
+        "location": "West Us",
+        "sku": {
+			"name": "Developer",
```
Comment (TianoMS) [inline tests/2016-07-07/Microsoft.ApiManagement.tests.json]: Please fix the indents in this file.

**Comment by TianoMS** [comment]
@solankisamir The changes look good to me. Just one format issue.

File: tests/2016-07-07/Microsoft.ApiManagement.tests.json (hunk 1 above)
Comment (solankisamir) [inline tests/2016-07-07/Microsoft.ApiManagement.tests.json]: Fixed the indentation issue with the file.

**Comment by rossedman** [comment]
Hooray! Would love to see this merged.

**Pull request merged**